Library/ViewModels/ProjectPamphletCreatorHeaderCellViewModel.swift

```diff
@@ -0,0 +1,81 @@
+import Foundation
+import KsApi
+import ReactiveSwift
+import UIKit
+
+public protocol ProjectPamphletCreatorHeaderCellViewModelInputs {
+  func configure(with project: Project)
+  func viewProgressButtonTapped()
+}
+
+public protocol ProjectPamphletCreatorHeaderCellViewModelOutputs {
+  var buttonTitle: Signal<String, Never> { get }
+  var launchDateLabelAttributedText: Signal<NSAttributedString, Never> { get }
+  var notifyDelegateViewProgressButtonTapped: Signal<Project, Never> { get }
+}
+
+public protocol ProjectPamphletCreatorHeaderCellViewModelType {
+  var inputs: ProjectPamphletCreatorHeaderCellViewModelInputs { get }
+  var outputs: ProjectPamphletCreatorHeaderCellViewModelOutputs { get }
+}
+
+public final class ProjectPamphletCreatorHeaderCellViewModel: ProjectPamphletCreatorHeaderCellViewModelType,
+  ProjectPamphletCreatorHeaderCellViewModelInputs, ProjectPamphletCreatorHeaderCellViewModelOutputs {
+  public init() {
+    self.buttonTitle = self.projectSignal
+      .map(title(for:))
+
+    self.launchDateLabelAttributedText = self.projectSignal
+      .map(attributedLaunchDateString(with:))
+      .skipNil()
+
+    self.notifyDelegateViewProgressButtonTapped = self.projectSignal
+      .takeWhen(self.viewProgressButtonTappedSignal.ignoreValues())
+  }
+
+  private let (projectSignal, projectObserver) = Signal<Project, Never>.pipe()
+  public func configure(with project: Project) {
+    self.projectObserver.send(value: project)
+  }
+
+  private let (viewProgressButtonTappedSignal, viewProgressButtonTappedObserver) =
+    Signal<Void, Never>.pipe()
+  public func viewProgressButtonTapped() {
+    self.viewProgressButtonTappedObserver.send(value: ())
+  }
+
+  public let buttonTitle: Signal<String, Never>
+  public let launchDateLabelAttributedText: Signal<NSAttributedString, Never>
+  public let notifyDelegateViewProgressButtonTapped: Signal<Project, Never>
+
+  public var inputs: ProjectPamphletCreatorHeaderCellViewModelInputs { return self }
+  public var outputs: ProjectPamphletCreatorHeaderCellViewModelOutputs { return self }
+}
+
+private func title(for project: Project) -> String {
+  return project.state == .live ? Strings.View_progress() : Strings.View_dashboard()
+}
+
+private func attributedLaunchDateString(with project: Project)
+  -> NSAttributedString? {
+  let date = Format.date(secondsInUTC: project.dates.launchedAt, dateStyle: .long, timeStyle: .none)
```

Comment:
We _might_ want to pass a timezone of UTC to this date formatter so that the date doesn't get converted to the device's timezone? 🤔